docs/src/installation.rst

```diff
@@ -3,46 +3,218 @@
+
+An example script for fetching the data manually (you can configure ``cron`` to run this weekly):
+
+.. code-block:: shell
+
+    #!/bin/bash
+
+    rm -f /tmp/WSRT_Measures.ztar
+    wget ftp://ftp.astron.nl/outgoing/Measures/WSRT_Measures.ztar -P /tmp -q
+    tar zxf /tmp/WSRT_Measures.ztar -C /var/lib/casacore/data
+    chmod -R 755 /var/lib/casacore/data
+
+Installing from source
+~~~~~~~~~~~~~~~~~~~~~~
+
+To install the backend on a different host system, or to install a custom version, you can build it from source from the `backend repository <https://github.com/CARTAvis/carta-backend/>`_ on GitHub. The `dockerfiles <https://github.com/CARTAvis/carta-backend/tree/dev/Dockerfiles>`_ in the backend repository are a good starting point for installing and configuring all the build dependencies on different Linux distributions.
+
+Once all dependencies have been installed, check out the backend repository with all its submodules, and build using ``cmake``.
+
+.. code-block:: shell
+
+    # Clone the backend repository
+    git clone --recurse-submodules https://github.com/CARTAvis/carta-backend.git
+    cd carta-backend
+
+    # Configure the build
+    mkdir build
+    cd build
+    cmake ..
 
-    The ``casacore-data`` package is recommended by the Ubuntu backend package, but installing it is optional. The packages in our PPA should be compatible both with the ``casacore-data`` package in the core Ubuntu repositories and with the package provided by the `Kern PPAs <https://launchpad.net/~kernsuite>`_. You may also wish to manage the required data files without using a package.
+    # Build
+    make -j8
 
-To install the backend on a different host system, or to install a custom version, you can build it from source from the `backend repository <https://github.com/CARTAvis/carta-backend/>`_ on GitHub.
+The backend executable will be located in the ``build`` directory.
+
+.. note::
+
+    Ensure that a custom ``processCommand`` is set in the controller configuration and that the backend path is updated in the sudoers file.
 
 .. _install_frontend:
 
 Installing the frontend
 -----------------------
 
-If you install the controller from NPM, the corresponding packaged version of the frontend will also be installed automatically. If you wish to install the controller from source, or would like to use a custom frontend version, you can install it from the `frontend repository <https://github.com/CARTAvis/carta-frontend/>`_ on GitHub.
+If you install the controller package from NPM, the corresponding packaged version of the frontend will be installed automatically as a dependency. However, you may wish to install a custom version of the frontend if you are installing the controller from source, or if you would like to test an updated frontend version.
+
+.. note::
+
+    Ensure that a custom ``frontendPath`` is set in the controller configuration.
```

Comment:
This note seems to be suggest that a `frontendPath` should always be set, though only needed if you want a diff-than-default for that version.

Reply:
I made the wording more explicit.